doc/doxygen/src/roadmap.md

```diff
@@ -9,85 +9,66 @@ The text and items below are tentative, up for discussion, to be updated by regu
 (contact/steering: [Martine](https://github.com/miri64))
 
 1. ICN stack support clean-up
-2. discuss mid- and long-term plans for network stack maintenance & development (GNRC vs other supported stacks)
-3. Support for CoAP blockwise transfer, observe. Also see [Ken's](https://github.com/kb2ma) unofficial [CoAP for RIOT](https://github.com/kb2ma/RIOT/wiki/CoAP-Home) page.
-4. revisit network time synchronization
+1. discuss mid- and long-term plans for network stack maintenance & development (GNRC vs other supported stacks)
+1. revisit network time synchronization
 
 
 
 # Network Stack Low layers
 (contact/steering: [Peter](https://github.com/PeterKietzmann))
 
-1. Towards fully open source support of 6LoWPAN over BLE
-2. Re-integrate and clean-up 802.15.4 TSCH integration
-3. Retransmissions by MAC: finalize and merge `netdev_retrans`
-4. Point-to-Point Protocol (PPP): finalize and merge `gnrc_ppp`
+1. Re-integrate and clean-up 802.15.4 TSCH integration
+1. Point-to-Point Protocol (PPP): finalize and merge `gnrc_ppp`
 
 
 
 # Power Modes
 (contact/steering: [Hauke](https://github.com/haukepetersen))
```

Comment:
Maybe @jue89 should be rather assigned here.

Reply:
I agree. But let's have a separate PR for this, as this PR would block otherwise.